modules/ROOT/pages/vm/vm-publish-listen.adoc

```diff
@@ -30,10 +32,5 @@ Suppose you have a flow that is triggered by an HTTP request with a JSON payload
         <logger/>
     </flow>
 ----
-+
-In this example you can see:
-+
-* The `publish` operation references the queue that you created in the other flow.
-* The `content` parameter allows you to use DataWeave to build the value to send (without producing side effects on the message). If you do not specify a `content`, the entire payload will be sent.
-* The `publish` operation is one-way. Once the message is posted into the queue, the flow continues. The processing on the listening side happen asynchronously. You do not receive notification of its outcome.
-* The `queueName` specified in the `publish` operation must have a matching `<vm:listener>` element that points to the same queue. If you attempt to publish to a queue for which no listener exists, an error will be raised. This behavior prevents message loss caused by sending messages into a queue that nobody is listening to.
\ No newline at end of file
+
+In this example, the `<vm:publish>` and `<vm:listener>` elements point to the same `queueName` queue. If you attempt to publish to a queue for which no listener exists, an error will raise. This behavior prevents message loss caused by sending messages into a queue that nobody is listening to.
```

Comment:
"In this example, the `<vm:publish>` and `<vm:listener>` elements point to the same queue. If you attempt to publish to a queue for which no listener exists, an error is raised."

It is not clear what "this behavior" refers to. Maybe say something like this:
"Having the  the `<vm:publish>` and `<vm:listener>` elements point to the same queue prevents message loss  caused by sending messages to a queue that nobody is listening to.